src/styles/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  height: 406px;
+  font-family: $base-font-family;
+  color: $primary-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid #F3F3F3;
+
+  .imac {
+    height: 134px;
+    width: 160px;
+    background-image: url(../images/imac.jpeg);
+    background-size: 160px 134px;
+    margin: 32px 19px 40px;
+    background-repeat: no-repeat;
+  }
+
+  .text {
+    padding: 0 16px;
+
+    .headline {
+      font-size: $secondary-font-size;
+      line-height: $primary-line-height;
+      margin: 0;
+      font-weight: 500;
+    }
+
+    .code {
+      font-size: $primary-font-size;
+      line-height: $secondary-line-height;
+      margin: 4px 0 16px;
```

Comment:
use paddings for entire card instead of setting horizontal margins for each element